Providers/Profile.swift

```diff
@@ -137,17 +125,12 @@ protocol Profile: AnyObject {
 
     func removeAccount()
 
-    func getClients() -> Deferred<Maybe<[RemoteClient]>>
-    func getCachedClients()-> Deferred<Maybe<[RemoteClient]>>
     func getClientsAndTabs() -> Deferred<Maybe<[ClientAndTabs]>>
-
     func getCachedClientsAndTabs(completion: @escaping ([ClientAndTabs]) -> Void)
     func getCachedClientsAndTabs() -> Deferred<Maybe<[ClientAndTabs]>>
 
     func cleanupHistoryIfNeeded()
 
-    func sendQueuedSyncEvents()
```

Comment:
Looks like `sendQueuedSyncEvents` is used in [profile.sendItem](https://github.com/mozilla-mobile/firefox-ios/blob/6ca9dc075f18131fd640683158cbbd2803b42db3/Providers/Profile.swift#L621